src/main/MantisProcess.ts

```diff
@@ -128,6 +128,7 @@ export const MantisProcess = (spawn: typeof childProcess.spawn) => (
           'mantis.datadir': mantisDataDir,
         },
         Object.entries,
+        array.map(([key, value]) => [key, isWin ? `"${value}"` : value]),
```

Comment:
Can't we enclose the values in quotation marks on every platform?

Reply:
Not ;( Was testing it on mac and Win simultanously